ql/lib/codeql/kaleidoscope/Cfg.qll

```diff
@@ -0,0 +1,254 @@
+private import codeql.kaleidoscope.Ast
```

Comment:
This file should probably be in `cfg/internal` instead, and then a simple user-facing layer here instead.

Reply:
The `Cfg::Make` module exposes both the user-facing `Cfg::Node` class as well as the classes that are used to construct the Cfg. It would be nice if it had a `Public` module that contains the classes and predicates that we would like to expose to query writers .